infra/resources/prod/main.tf

```diff
@@ -232,3 +232,17 @@ module "cosmos" {
     instance_number = "01"
   }
 }
+
+module "io_com_storage_account" {
+  source              = "../_modules/com_storage_account/"
+  resource_group_name = data.azurerm_resource_group.itn_messages.name
```

Comment:
we should use `itn_com` resource group (`io-p-itn-com-rg-01`) for new resources

Reply:
[8640ea0](https://github.com/pagopa/io-messages/pull/107/commits/8640ea092a73f233ba5b83b69acbb1ce730c596e)